pkg/gui/whitespace-toggle.go

```diff
@@ -9,5 +13,5 @@ func (gui *Gui) toggleWhitespaceInDiffView() error {
 	}
 	gui.c.Toast(toastMessage)
 
-	return gui.refreshFilesAndSubmodules()
+	return gui.c.Refresh(types.RefreshOptions{Mode: types.ASYNC})
```

Comment:
This is probably too big of a hammer, I guess we could be more considerate as to which views we refresh. But it's easy, and [there's precedence for this](https://github.com/jesseduffield/lazygit/blob/master/pkg/gui/diffing.go#L119). 😄

Reply:
Come to think of it, we shouldn't actually need to refresh anything. We should just be able to re-trigger focusing the selected list item.

See if this does the trick: 
```go
gui.currentSideListContext().HandleFocus(types.OnFocusOpts{})
```